src/app/db/tables.py

```diff
@@ -75,7 +75,7 @@ class EventType(str, enum.Enum):
     Column("id", Integer, primary_key=True),
     Column("login", String(50)),
     Column("owner_id", Integer, ForeignKey("users.id")),
-    Column("access_id", Integer, ForeignKey("accesses.id"), unique=True),
+    Column("access_id", Integer, ForeignKey("accesses.id", ondelete="CASCADE"), unique=True),
```

Comment:
So deleting the device will delete the access? If so, shouldn't we do the same for the user?

Reply:
And shall we do it also for the owner_id ?  If we delete the users, the devices will disappear?

Reply:
Yes I thought that I also had added the on delete cascade on user but I forgot.

I figured out that it is the opposite: deleting the access deletes the device/user. I am working on fixing the device/user deletion. 

I will add the mechanism owner id as well !

Reply:
Another issue when we use the on cascade mechanism -> as it perform at db level we have to find a way to mock it in our unit tests